src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -4608,35 +4608,6 @@ void M()
 o1 OPERATOR s2, //-ObjectKIND
 o1 OPERATOR o2  //-ObjectKIND" + Postfix;
 
-        private const string PostfixIncrementTemplate = Prefix + @"
```

Comment:
>PostfixIncrementTemplate [](http://example.com/codeflow?start=29&length=24)

Consider keeping (and perhaps comment out?) since there is a commented out reference in `TestOperatorOverloadResolution()`.